symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php

```diff
@@ -32,12 +40,38 @@ public function getAddons()
     }
 
     /**
-     * @return array
+     * @return mixed|string
+     * @throws CoreServiceException
      */
-    protected function getAddonsFromMP()
+    private function getAddonsFromMP()
     {
-        $addons = array();
-        return $addons;
+        $token = $this->getApiToken();
+        if ($token == 'Network Error') {
+            return $token;
+        }
+        $headers = array(
+            'Accept' => 'application/json',
+            'Authorization' => 'Bearer ' . $token
+        );
+
+        try {
+            $response = $this->getApiClient()->get('/api/v1/addon',
```

Comment:
pls dont hardcode the endpoints .. define it as constants and use it. we may need to change in the future

Reply:
corrected